tasks/bloom-etl/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Runner, Extractor, Transformer, Loader, defaultMap } from "./src/etl"
+import { DbConfig, Jurisdiction, UrlInfo } from "./src/types"
+import { knex } from "knex"
+import axios from "axios"
+
+// No need to retrieve these dynamically since they never change, per Exygy
+const jurisdictions: Array<Jurisdiction> = [
```

Comment:
Can we put the jurisdictions into the env variables as well ?if we are putting the api endpoint we should make it clear that these variables need to change too if that url changes -- OR we follow the pattern in Bloom and dynamically fetch these via /jurisdictions and use the jurisdiction names

Reply:
I thought about dynamically fetching them, but Morgan said the list of jurisdictions were static and would never change, so I hardcoded them to make it more clear and explicit about what jurisdictions are being used.  Honestly, I didn't even add an env var for the endpoint until the last minute, and that's mainly because I thought there might be a need to bypass the proxy in rare occasions.  Env vars don't do well with structured data and I like having the jurisdiction names for documentation and more useful log messages, so I think we want to keep all of that in.  

Keep in mind that this code will only have a lifetime of 6-12 months before it isn't needed any longer, so if this is really something that's worth investing the eng effort into, it might be worth looping back on the discussion, aligning on a direction, and addressing it in a new PR